docs/sources/mimir/release-notes/v2.13.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Grafana Mimir version 2.13 release notes"
+menuTitle: "V2.13 release notes"
+description: "Release notes for Grafana Mimir version 2.13"
+weight: 1100
+---
+
+# Grafana Mimir version 2.13 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.13 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Store-gateway can be configured to explicitly disable or enable tenants**. This is useful in cases when compaction slows down for a tenant to temporarily exclude a tenant while compaction catches up without impact other tenants.
```

Comment:
Is this the most exciting new feature? I'd expect us to put the top highlights first.

Reply:
haha, good point. This was largely ordered as the changelog

Reply:
Looks reasonable to me, @zhehao-grafana might have opinions though.